### ISSUE TITLE
[a11y] Issue 33: No focus on text block when reading "Remaining Statements"

### ISSUE THREAD

**Issue by kk-slider18** [NewJerseyStyle/polis#129]
**Description:**
When the screen reader reaches the "Remaining Statements" section, it does not set focus on the text block.

**Component/Area:** Open Participation Page

**Source:** Screen Reader

**Affected Elements:** Remaining Statements section

**Comment by cplkake** [comment]
I have set this to P0. The remaining statements section informs users how many statements are left to vote for, a key piece of information.

**Comment by SolomonHsu** [comment]
Hey all,
Yumi reached out asking me for some support on this. Leaving a comment here for everyone's awareness.
When focusing on elements, the keyboard focus should only be on the interactive elements. Examples of interactive elements are:
- links
- buttons
- form fields
- scrollable areas
- popup modals or small popups

In the event of the area is a text area, the text area itself should not receive a tabindex="0" to allow focus.
Reference: https://webaim.org/techniques/keyboard/
